### DISCUSSION TITLE
Support for macOS-14 (ARM64 builds and tests)

### DISCUSSION THREAD

**Discussion by cbrnr** [pypa/cibuildwheel#1747] [Q&A] [answered]
Now that macOS ARM64 runners are publicly available (https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/), I was wondering how this image can be used by cibuildwheel to build and test for this platform. Thanks!

Update: Apparently, just specifying the image does not work, since `pipx` is missing (and maybe other things too): https://github.com/cbrnr/sleepecg/actions/runs/7724128496/job/21055666712?pr=206

**Comment by henryiii** [comment, marked as answer]
Use cibuildwheel 2.16.5, which was released yesterday and works around the pipx issue. Also see #1746, #1745.
